Source/JavaScriptCore/jit/AssemblyHelpers.cpp

```diff
@@ -1437,6 +1437,35 @@ void AssemblyHelpers::emitSaveOrCopyLLIntBaselineCalleeSavesFor(CodeBlock* codeB
     spooler.finalizeGPR();
 }
 
+void AssemblyHelpers::checkArity(VM& vm, unsigned numberOfParameters, GPRReg argumentCountIncludingThisGPR, GPRReg paddingOutputGPR, GPRReg scratchGPR0, GPRReg scratchGPR1, JumpList& stackOverflow)
```

Comment:
I wonder if we should name this something like "getArityPadding"

Reply:
my reasoning is this, we've already checked arity at this point. And the code is assuming that (not checking for overflow in sub)

Reply:
Changed.